test/jasmine/tests/gl2d_pointcloud_test.js

```diff
@@ -210,4 +210,49 @@ describe('pointcloud traces', function() {
         .catch(failTest)
         .then(done);
     });
+
+    it('@gl should respond to drag', function(done) {
+        function _drag(p0, p1) {
+            mouseEvent('mousemove', p0[0], p0[1], {buttons: 1});
+            mouseEvent('mousedown', p0[0], p0[1], {buttons: 1});
+            mouseEvent('mousemove', (p0[0] + p1[0]) / 2, (p0[1] + p1[1]) / 2, {buttons: 1});
+            mouseEvent('mousemove', p1[0], p1[1], {buttons: 0});
+            mouseEvent('mouseup', p1[0], p1[1], {buttons: 0});
+        }
+
+        function _assertRange(msg, xrng, yrng) {
+            expect(gd._fullLayout.xaxis.range).toBeCloseToArray(xrng, 2, msg);
+            expect(gd._fullLayout.yaxis.range).toBeCloseToArray(yrng, 2, msg);
+        }
+
+        Plotly.plot(gd, Lib.extendDeep({}, plotData))
+        .then(delay(40))
+        .then(function() {
+            _assertRange('base', [-0.548, 9.548], [-1.415, 10.415]);
+        })
+        .then(function() { _drag([200, 200], [350, 350]); })
+        .then(delay(400))
```

Comment:
Wondering if we really need `400` delay here? The test passed with `40` on my machine.

Reply:
Ha good call! I forget to change this back to `40`. Done in -> https://github.com/plotly/plotly.js/pull/3647/commits/bc87c01692d93613bb6a56657901f1f1a6d4cd82